src/PaginationNav/PaginationNav.svelte

```diff
@@ -76,7 +82,11 @@
       <Button
         kind="ghost"
         tooltipAlignment="center"
-        tooltipPosition="bottom"
+        tooltipPosition="{tooltipPosition === 'inside'
```

Comment:
Since this passes `tooltipPosition` to `Button`, perhaps all we need is
`tooltipPosition="{tooltipPosition}"`. I don't think `outside` or `inside` has
intuitive meaning, so we can leave out those options.

Everything else looks good 👍

Reply:
I wasn't going to push back, but then I realised: without `inside` and `outside` there is no way to achieve that layout without adding an additional prop like `leftTooltipPosition` and `rightTooltipPosition` to the `PagionationNav` component.

I haven't actually used this component in production before, but I can see times where constrained layouts may force the need for the inside and outside options.

![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/10446022/aa9ed7d0-a1db-476e-af6b-33384dd2d332)

![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/10446022/e1b0640f-0217-4680-a7de-eb0869b680d1)

What do you think?

Reply:
Looking at this code:

```html
<Button
        tooltipPosition="{tooltipPosition === 'inside'
          ? 'right'
          : tooltipPosition === 'outside'
          ? 'left'
          : tooltipPosition}"
>
```

When `tooltipPosition` is `inside`, it will set `tooltipPosition="right"`.
When it is `outside`, then it sets `tooltipPosition="left"`. 

Since the underlying effect is `right` or `left`, I'm not sure how aliasing that with `outside` and `inside` can be used. 

>I can see times where constrained layouts may force the need for the inside and outside options.

Can you provide an example?

Reply:
Did the images not come through in my previous message demoing "outside"?

If the dev passes in "left" **or** "right" they get both labels on the same side. There's no way for the dev to pass in "left" **and** "right".